kpi/README.rst

```diff
@@ -0,0 +1,131 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===
+KPI
+===
+
+This module provides the basis for creating key performance indicators,
+including static and dynamic thresholds (SQL query or Python code),
+on local and remote data sources.
+
+The module also provides the mecanism to update KPIs automatically.
+A scheduler is executed every hour and updates the KPI values, based
+on the periodicity of each KPI. KPI computation can also be done
+manually.
+
+A threshold is a list of ranges and a range is:
+
+* a name (like Good, Warning, Bad)
+* a minimum value (fixed, sql query or python code)
+* a maximum value (fixed, sql query or python code)
+* color (RGB code like #00FF00 for green, #FFA500 for orange, #FF0000 for red)
+
+
+Installation
+============
+
+No installation required.
+
+Configuration
+=============
+
+Users must be added to the appropriate groups within Odoo as follows:
+* Creators: Settings > Users > Groups > Management System / User
+* Responsible Persons: Settings > Users > Groups > Management System / Approving User
+
+Usage
+=====
+
+To use this module:
+
+* Go to Management System > Nonconformities
+* Click on Create to enter the following information:
+
+* Partner : Customer, supplier or internal personnel
+* Related to: Any reference pointing to the NC (order id, project id, task id, etc.)
+* Responsible: Person responsible for the NC
+* Manager : Person managing the department or owner of the procedure
+* Filled in by: Originator of NC report
+* Origins:  The source of the NC, how was it discover
+* Procedures:  Against which procedure is the NC
+* Description: Evidence, reference to the standards
+
+* Click on Save and then on Send for Analysis.
+
+As an approving user, go to the newly created NC and fill in the following
+information in the tab named Causes and Analysis:
+
+* Causes: Add root causes
+* Analysis: Describe the results of the investigation
+* Severity: Select the severity among unfounded, minor and major
+* Immediate action: Create or select an immediate action if appropriate
+
+Click on Approve and then on Send for Review.
+
+In the Actions tab, select or create new actions by entering the following
+items:
+
+* Subject: What must be done - Return to Supplier, Use As Is, Scrap, Rework,
+  Re-grade, Repair
+* Deadline: Date by which the action must be completed
+* Responsible: Person in charge for implementing the action
+* Type: Immediate, corrective or preventive actions or improvement opportunity
+* Description: Details of the action
+
+When the action is created, a notification is sent to the person responsible
+for the action.
+
+When the action plan is reviewed (comments) and approved, each action of the
+plan is opened.
+
+When all actions of the plan are done, their effectiveness must be evaluated
+before closing the NC.
+
```

Comment:
@naousse Replace this usage section (from the nonconformity module) with the link to the video I sent you.